packages/devtools-network-console/src/ui/RequestBody/index.tsx

```diff
@@ -57,33 +56,27 @@ interface IConnectedProps {
 export type IRequestBodyEditorProps = IOwnProps & IConnectedProps;
 const BODY_CONTENT_TYPES = [{
     key: 'text/plain',
-    text: 'Plain text (text/plain)',
-    iconProps: { iconName: 'TextDocument' },
+    text: 'text/plain',
 },
 {
     key: 'application/json',
-    text: 'JSON (application/json)',
-    iconProps: { iconName: 'Script' },
+    text: 'application/json',
 },
 {
     key: 'text/xml',
-    text: 'XML (text/xml)',
-    iconProps: { iconName: 'ChevronLeftMed' },
+    text: 'text/xml',
 },
 {
     key: 'application/xml',
-    text: 'XML (application/xml)',
-    iconProps: { iconName: 'ChevronRightMed' },
+    text: 'application/xml',
 },
 {
     key: 'text/html',
-    text: 'HTML (text/html)',
-    iconProps: { iconName: 'FileHTML' },
+    text: 'text/html',
 },
 {
     key: 'application/javascript',
-    text: 'JavaScript (application/javascript)',
-    iconProps: { iconName: 'JavaScriptLanguage' },
```

Comment:
no more icons?

Reply:
Correct. We removed the library and the only things we would have kept were the icons.